### PR TITLE
roo.me README file

### PR DESCRIPTION
**Pull request by AmitAharoni**
# Description
Update roo.me README  file to his final form.
Added Text, photos, links and more. see `screenshots` section.

## Screenshots
**NOTE: These are screenshots so the format and flow may appear broken!**

image

image
image
image
image
image

**NOTE: These are screenshots so the format and flow may appear broken!**

### Issues closed by this PR
#285.

### PR Dependencies
NONE!

### CODE REVIEW

**Review by nadavsu — DISMISSED** [review]
Readme should not contain screenshots as text.
Also, I don't like the CLICK HERE.
I think the previous readme was good as it was, maybe just add the demo link and that's it.

**Comment by nadavsu** [comment]
Also, if you want to have a nice Contributors section, check out the contributors section in [this README](https://github.com/techx/quill/blob/master/README.md#contributing) and try to copy it.

**Review by tamirho — DISMISSED** [review]
> Readme should not contain screenshots as text.

I agree with @nadavsu .. we need to use the benefits of `MD` file and not use screenshots to view text.
I think that the readme file should be clean and easy to use and navigate.
Maybe use the current design and update the text in the relevant sections.

**Comment by AmitAharoni** [comment]
Got it, I agree.
Will do.

@nadavsu @tamirho

**Comment by AmitAharoni** [comment]
@tamirho @nadavsu 
Updated.
Please review again.

File: README.md [hunk 1]
```diff
@@ -1,81 +1,68 @@
-![small-white](https://user-images.githubusercontent.com/53123142/110364209-287bbd80-804c-11eb-91c3-a158b115ef27.png)
+<img src="https://user-images.githubusercontent.com/53123142/110364209-287bbd80-804c-11eb-91c3-a158b115ef27.png" width="650">
 
-roo.me is an open source web-application for easily finding roommates. 
 
-It works by showing users the best possible matches based on their profile information and preferences, with an easy 'yes or no' interface.
+# Description
+roo.me is an interactive and easy swipe web-app for finding roommate apartments.
```
Comment (DanielMy777) [inline README.md]: Looks good!
maybe use something else instead of 'swipe' as we don't do that lol

**Review by DanielMy777 — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (AmitAharoni) [reply README.md]: OK

**Review by DanielMy777 — APPROVED** [review]
LGTM Good job!

**Pull request merged**